generate_tuxsuite.py

```diff
@@ -43,6 +43,8 @@ def emit_tuxsuite_yml(config, tree, llvm_version):
         print("# Invoke locally via:")
         print(f"# $ git clone -b {ref} --depth=1 {repo} linux")
         if patches_flag:
+            # Input: '--patch-series ... '
+            # Output: '...'
             patches_folder = patches_flag.strip().rsplit(' ', 1)[-1]
```

Comment:
does
```diff
-patches_folder = patches_flag.strip().rsplit(' ', 1)[-1]
+patches_folder = patches_flag.split(' ')[1]
```
work here?

Reply:
Sure, I do not see why not, I can change it to that if you prefer.

Reply:
yes please

Reply:
Done: e58a2daf8e53386c418025da7ac1c8ad7be6b9e6